upgrade_schema_test.py

```diff
@@ -274,7 +274,7 @@ def test_upgrade_schema(self):
         except NotFoundException:
             pass
         thrift_client.system_add_keyspace(
-            KsDef(ks_name, 'org.apache.cassandra.locator.SimpleStrategy', {'replication_factor': '1'}, cf_defs=[]))
+            KsDef(ks_name, 'org.apache.cassandra.locator.NetworkTopologyStrategy', {'replication_factor': '1'}, cf_defs=[]))
```

Comment:
```suggestion
            KsDef(ks_name, 'org.apache.cassandra.locator.NetworkTopologyStrategy', {'replication_factor': 1}, cf_defs=[]))
```

Reply:
there are many places we use str instead of int like this - I was wondering if we want to touch that if it works...

Reply:
just to align, as you have changed few of these strings into integers already...

Reply:
but i'm personally fine with it

Reply:
I don't know if I would touch this file, we are not using it since forever, I'm not sure we have any job pointing to it.

We have been trying to deprecate the thrift protocol for a while now (it's now disabled by default, so these tests haven't run since a long long time ago)

Reply:
I didn't analyze much when replacing it, I did it semi-automatic way. Probably there are many more places we don't use anymore...